src/pinnwand/static/pinnwand.js

```diff
@@ -167,6 +167,12 @@ function setupShowPage() {
         });
     };
 
+    let shareButton = document.getElementById("copyurl-btn");
+
```

Comment:
I think this needs a `null` check or coalesce before continuing into the event listener below.